### PR TITLE
Improve .NET 5 naming

### PR DESCRIPTION
**Pull request by gewarren**
.NET 5.0 -> .NET 5

Contributes to #26777.

### CODE REVIEW

File: docs/core/compatibility/aspnet-core/5.0/kestrel-libuv-transport-obsolete.md [hunk 1]
```diff
@@ -8,9 +8,9 @@ ms.date: 10/01/2020
 
 Earlier versions of ASP.NET Core used Libuv as an implementation detail of how asynchronous input and output was performed. In ASP.NET Core 2.0, an alternative, <xref:System.Net.Sockets.Socket>-based transport was developed. In ASP.NET Core 2.1, Kestrel switched to using the `Socket`-based transport by default. Libuv support was maintained for compatibility reasons.
 
-At this point, use of the `Socket`-based transport is far more common than the Libuv transport. Consequently, Libuv support is marked as obsolete in .NET 5.0 and will be removed entirely in .NET 6.0.
+At this point, use of the `Socket`-based transport is far more common than the Libuv transport. Consequently, Libuv support is marked as obsolete in .NET 5 and will be removed entirely in .NET 6.0.
```
Comment (tommcdon) [inline docs/core/compatibility/aspnet-core/5.0/kestrel-libuv-transport-obsolete.md:11]: Should we also change 6.0 to 6?

File: docs/core/compatibility/aspnet-core/5.0/blazor-rendertreeframe-fields-become-properties.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ This change won't affect many developers because:
 
 * Any app or library that simply uses `.razor` files (or even manual <xref:Microsoft.AspNetCore.Components.Rendering.RenderTreeBuilder> calls) to define its components wouldn't be referencing this type directly.
 * The `RenderTreeFrame` type itself is regarded as an implementation detail, not intended for use outside of the framework. ASP.NET Core 3.0 and later includes an analyzer that issues compiler warnings if the type is being used directly.
-* Even if you reference `RenderTreeFrame` directly, this change is binary-breaking but not source-breaking. That is, your existing source code will compile and behave properly. You'll only encounter an issue if compiling against a .NET Core 3.x framework and then running those binaries against the .NET 5.0 RC1 and later framework.
+* Even if you reference `RenderTreeFrame` directly, this change is binary-breaking but not source-breaking. That is, your existing source code will compile and behave properly. You'll only encounter an issue if compiling against a .NET Core 3.x framework and then running those binaries against the .NET 5 RC1 and later framework.
```
Comment (tommcdon) [inline docs/core/compatibility/aspnet-core/5.0/blazor-rendertreeframe-fields-become-properties.md:15]:
```suggestion
* Even if you reference `RenderTreeFrame` directly, this change is binary-breaking but not source-breaking. That is, your existing source code will compile and behave properly. You'll only encounter an issue if compiling against a .NET Core 3.x framework and then running those binaries against the .NET 5 and later framework.
```

**Review by tommcdon — APPROVED** [review]
Looks good to me, left a couple minor suggestions for your review

File: docs/core/compatibility/aspnet-core/5.0/kestrel-libuv-transport-obsolete.md (hunk 1 above)
Comment (gewarren) [reply docs/core/compatibility/aspnet-core/5.0/kestrel-libuv-transport-obsolete.md:11]: I should have mentioned - .NET 6 is covered in #26842.

**Review by danroth27 — APPROVED** [review]
LGTM

**Review by BillWagner — APPROVED** [review]
This LGTM as well.

**Pull request merged**